docs/source/docker/getting_started/first_steps.rst

```diff
@@ -509,7 +509,7 @@ The output directory is structured in the following way:
 
    * information about the job (duration, processed files etc.)
    * estimated savings in terms of labeling costs and CO2 due to the smaller dataset
-   * statistics about the dataset before and after sampling
+   * statistics about the dataset before and after selection
```

Comment:
and after selection -> and after applying the selection strategy

Reply:
I used `before/after selection process` for these cases.